client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx

```diff
@@ -10,33 +10,31 @@ import i18n from 'lib/mixins/i18n';
 import PurchaseDetail from './purchase-detail';
 
 const BusinessPlanDetails = ( { isFreeTrial, selectedSite } ) => {
-	const showGetFreeDomainTip = ! isFreeTrial;
-
 	return (
 		<div>
-			{ showGetFreeDomainTip
+			{ isFreeTrial
 			? <PurchaseDetail
-					additionalClass="get-free-domain"
-					title={ i18n.translate( 'Get your custom domain' ) }
-					description={
-						i18n.translate(
-							"Replace your site's address, {{em}}%(siteDomain)s{{/em}}, with a custom domain. " +
-							'A free domain is included with your plan.',
-							{
-								args: { siteDomain: selectedSite.domain },
-								components: { em: <em /> }
-							}
-						)
-					}
-					buttonText={ i18n.translate( 'Claim your free domain' ) }
-					href={ '/domains/add/' + selectedSite.slug } />
+				additionalClass="live-chat"
```

Comment:
Do we still want to promote live chat in this case? It's not in the doc, so we should probably ask @mikeshelton1503.